make/build.py

```
@@ -1,26 +1,32 @@
+
+from . import (
+    REPO,
+    base_images,
+    image_exists,
+    image_labels,
+    is_root,
+    podman,
+    podman_cmd,
+)
+from .config import parse_config
+from .hash import hash
```

Ruff (0.15.9)

[error] Import `hash` is shadowing a Python builtin

(A004)